os/arch/arm/src/Makefile

```diff
@@ -91,6 +91,10 @@ else
 endif
 endif
 
+ifeq ($(CONFIG_ARCH_BOARD_CY4390X),y)
+	CFLAGS+=-I$(TOPDIR)/../external/WICED/bcm4390x/include
```

Comment:
Let's move this at Make.defs of chip or Makefile of board.

Reply:
it will be changed as your guidance.

BR, HJ Lee.